src/precice/tests/ParallelTests.cpp

```diff
@@ -47,7 +47,7 @@ struct ParallelTestFixture : testing::WhiteboxAccessor {
 BOOST_AUTO_TEST_SUITE(PreciceTests)
 BOOST_FIXTURE_TEST_SUITE(Parallel, ParallelTestFixture)
 
-
+/*
```

Comment:
Why is this test altered and removed? It should still work, right? It is very similar to the tests below, but still could be good to keep it as below we will probably test the new BB initialization and here the old one.